Core/HLE/sceNet.cpp

```diff
@@ -232,10 +115,6 @@ static void __ResetInitNetLib() {
 
 void __NetInit() {
 	portOffset = g_Config.iPortOffset;
-	if (PPSSPP_ID == 0) // For persistent ID/IP, New instances may have a possibility to have the same ID/IP if PPSSPPIDCleanup() being called on every reset
-	{
-		PPSSPP_ID = getPPSSPPInstanceNumber(); // This should be called when program started instead of when the game started/reseted
-	}
 
 	net::Init();
```

Comment:
I also need to remove this `net::Init()` from `__NetInit()` while rebasing because `net::Shutdown()` have been removed from `__NetShutdown()` so the number of WSAStartup and WSAShutdown will match.

Reply:
Ah, okay.